pom.xml

```diff
@@ -309,6 +309,29 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
It seems there are many things here that appear as change although they were committed previously... Did you merge from dev?